inbox/api/ns_api.py

```diff
@@ -532,6 +533,14 @@ def folders_labels_create_api():
     valid_display_name(g.namespace.id, category_type, display_name,
                        g.db_session)
 
+    if g.namespace.account.provider not in ['gmail', 'eas']:
+        # Translate the name of the folder to an actual IMAP name
+        # (e.g: "Accounting/Taxes" becomes "Accounting.Taxes")
+        display_name = imap_folder_path(
```

Comment:
Makes sense, but can you check what happens if you pass an IMAP path to `imap_folder_path`?